src/java.base/share/classes/sun/launcher/resources/launcher_zh_CN.properties

```diff
@@ -24,22 +24,25 @@
+相关的安全设置并继续\n    -XshowSettings:system\n                      （仅 Linux）显示主机系统或容器\n                      配置并继续\n    -Xss<size>        设置 Java 线程堆栈大小\n                      实际大小可以舍入到\n                      操作系统要求的系统页面大小的倍数。\n    -Xverify          设置字节码验证器的模式\n                      请注意，选项 -Xverify:none 已过时，\n                      可能会在未来发行版中删除。\n    --add-reads <module>=<target-module>(,<target-module>)*\n                      更新 <module> 以读取 <target-module>，而无论\n                      模块如何声明。 \n                      <target-module> 可以是 ALL-UNNAMED，将读取所有未命名\n                      模块。\n    --add-exports <module>/<package>=<target-module>(,<target-module>)*\n                      更新 <module> 以将 <package> 导出到 <target-module>，\n                      而无论模块如何声明。\n                      <target-module> 可以是 ALL-UNNAMED，将导出到所有\n                      未命名模块。\n    --add-opens <module>/<package>=<target-module>(,<target-module>)*\n                      更新 <module> 以在 <target-module> 中打开\n                      <package>，而无论模块如何声明。\n    --limit-modules <module name>[,<module name>...]\n                      限制可观察模块的领域\n    --patch-module <module>=<file>({0}<file>)*\n                      使用 JAR 文件或目录中的类和资源\n                      覆盖或增强模块。\n    --source <version>\n                      设置源文件模式中源的版本。\n    --finalization=<value>\n                      控制 JVM 是否执行对象最终处理，\n                      其中 <value> 为 "enabled" 或 "disabled" 之一。\n                      默认情况下，最终处理处于启用状态。\n    --sun-misc-unsafe-memory-access=<value>\n                      允许或拒绝使用不受支持的 API sun.misc.Unsafe\n                      <value> 为 "allow"、"warn"、"debug" 或 "deny" 之一。\n                      默认值为 "warn"。\n\n这些额外选项如有更改, 恕不另行通知。\n
 
 # Translators please note do not translate the options themselves
 java.launcher.X.macosx.usage=\n以下选项是特定于 macOS 的选项：\n    -XstartOnFirstThread\n                      在第一个 (AppKit) 线程上运行 main() 方法\n    -Xdock:name=<application name>\n                      覆盖停靠栏中显示的默认应用程序名称\n    -Xdock:icon=<path to icon file>\n                     覆盖停靠栏中显示的默认图标\n\n
 
+# Translators please note do not translate the options themselves
+java.launcher.opt.concise.header  =   用法：java [java options...] <application> [application arguments...]\n\n其中，<application> 为以下项之一：\n  <mainclass>                执行编译的主类的 main 方法\n  -jar <jarfile>.jar         执行 JAR 档案的主类\n  -m <module>[/<mainclass>]  执行模块的主类\n  <sourcefile>.java          编译和执行源文件程序\n\n其中，主要的 java 选项包括：\n  --class-path <class path>\n    其中，<class path> 是用于搜索类文件的目录和 JAR 档案列表（以 "{0}" 分隔）\n  --module-path <module path>\n    其中，<module path> 是用于搜索模块的目录和 JAR 档案列表（以 "{0}" 分隔）\n  -version\n    将产品版本输出到错误流并退出\n\n获取有关用法的其他帮助：           java --help\n获取交互式 Java 环境：    jshell
```

Comment:
It would be nicer if the  `java` and `jshell` could be aligned in the last section, but maybe it is not possible?
```
用法：java [java options...] <application> [application arguments...]

其中，<application> 为以下项之一：
  <mainclass>                执行编译的主类的 main 方法
  -jar <jarfile>.jar         执行 JAR 档案的主类
  -m <module>[/<mainclass>]  执行模块的主类
  <sourcefile>.java          编译和执行源文件程序

其中，主要的 java 选项包括：
  --class-path <class path>
    其中，<class path> 是用于搜索类文件的目录和 JAR 档案列表（以 ":" 分隔）
  --module-path <module path>
    其中，<module path> 是用于搜索模块的目录和 JAR 档案列表（以 ":" 分隔）
  -version
    将产品版本输出到错误流并退出

获取有关用法的其他帮助：           java --help
获取交互式 Java 环境：    jshell
```